docusaurus/docs/iOS/client/llc-client-overview.md

```diff
@@ -13,6 +13,8 @@ The `StreamChat` framework comes with controllers and delegates that you can use
 
 Each controller exposes API functionality and supports delegation. Controllers and their delegates are documented here based on the kind of data they control and allow you to observe. You can find examples on how to build your own view as well.
 
+In addition to controllers, we provide a modern [state layer](state-layer/state-layer-overview.md) with async-await and observable state objects.
+
```

Comment:
I think that this might get overlooked easily. 🤔 IMO, we should have a new section on top of the `Accessing State with Controllers`, with something like `Accessing State with the new State Layer`. And we should have a not that this is new and only available after 4.XX.0+